include/ts/apidefs.h.in

```diff
@@ -462,7 +462,9 @@ typedef enum {
   TS_EVENT_SSL_CERT                             = 60203,
   TS_EVENT_SSL_SERVERNAME                       = 60204,
   TS_EVENT_SSL_SERVER_VERIFY_HOOK               = 60205,
-  TS_EVENT_SSL_VERIFY_CLIENT                    = 60206
+  TS_EVENT_SSL_VERIFY_CLIENT                    = 60206,
+  TS_EVENT_VCONN_CONTINUE                       = 60207,
+  TS_EVENT_VCONN_ERROR                          = 60208
```

Comment:
Why can't TS_EVENT_CONTINUE and TS_EVENT_ERROR (generic) be used?  Or, why do we need an event at all, why not:

tsapi void TSVConnErrorTermination(TSVConn sslvcp);

instead of:

tsapi void TSVConnReenableEx(TSVConn sslvcp, TSEvent event);

Reply:
Hmmm.  There is a already a TS_EVENT_ERROR/CONTINUE.  I was following the pattern of TSHttpTxnReenable/TSHttpSsnReenable with TS_EVENT_HTTP_ERROR/CONTINUE.

We could use TS_EVENT_ERROR/CONTINUE instead of adding another pair of events.

I do like the Reenable pattern with a general event argument rather than having an explicit error reenable method.  This gives you the option of inserting different events to impact processing in the future.